src/components/NotificationRow.tsx

```diff
@@ -1,9 +1,13 @@
 import { FC } from "react";
 import { NotificationConsumer, Row } from "@canonical/react-components";
 
-const NotificationRow: FC = () => {
+interface Props {
+  className?: string;
+}
+
+const NotificationRow: FC<Props> = ({ className }) => {
```

Comment:
In this case we are trying to target the `row` element that is the wrapping element for `NotificationRow`, see below html example.  Since it is possible to have other `row` elements inside the panel, I thought it would be better to have a more specific class selector. Another way to solve this is by assuming notification will always appear at the top of a panel and therefore we can use a `:first-child` selector, but I'm afraid this approach might not be 100% accurate. 

```
<div class="panel-notification row">
  <div>
    <div class="p-notification--negative">
      <div class="p-notification__content">
        <h5 class="p-notification__title" data-testid="notification-title">
          Fail!
        </h5>
        <p class="p-notification__message"> </p>
        <button
          class="p-notification__close"
          data-testid="notification-close-button"
        >
          Close notification
        </button>
      </div>
    </div>
  </div>
</div>
```